### PR TITLE
Support more recent cuda versions 

### PR DESCRIPTION
**Pull request by abergeron**
Also make sure to not always report errors for nvrtc64_70

### CODE REVIEW

**Comment by abergeron** [comment]
This should get some testing on windows before it's merged.

**Comment by nouiz** [comment]
jenkins test windows

**Comment by abergeron** [comment]
Jenkins is fine, but I meant test on a windows machine that has cuda10 installed.

**Comment by nouiz** [comment]
jenkins didn't ran... It have problems.
I'll test locally here tomorrow.

File: src/gpuarray_buffer_cuda.c [hunk 1]
```diff
@@ -190,7 +190,8 @@ static int setup_lib(void) {
       } while (res != GA_NO_ERROR && i < versions_length);
     }
     if (res != GA_NO_ERROR)
-      return res;
+      // Return the error from the original attempt
+      return load_libnvrtc(major, minor, global_err);
```
Comment (nouiz) [inline src/gpuarray_buffer_cuda.c]: This is not the original major/minor that you try again to generate the original error. But the guessed major/minor.

Comment (abergeron) [reply src/gpuarray_buffer_cuda.c]: That's true.

Comment (abergeron) [reply src/gpuarray_buffer_cuda.c]: Should be fixed now.

**Comment by nouiz** [comment]
as jenkins isn't working, I tested locally and it work. So merging.

**Comment by nouiz** [comment]
Note, a new release could lower the number of questions we have.

**Pull request merged**